docs/conf.py

```diff
@@ -12,12 +12,14 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
+import django
 import os
+import sphinx_bootstrap_theme
```

Comment:
[Third party imports (here, `django` and `sphinx_bootstrap_theme`) should come after standard library imports](https://www.python.org/dev/peps/pep-0008/#imports).
```suggestion
```